lib/deck.rb

```diff
@@ -0,0 +1,56 @@
+require './lib/card'
+require 'pry'
+
+class Deck
+
+    attr_reader :cards
+
+    def initialize(cards)
+
+        @cards = []
+        @cards = cards
+    
+    end
+
+    def rank_of_card_at(index)
+        @cards[index].rank
+    end
+
+    def high_ranking_cards
+
+        high_rank_cards = []
+
+        @cards.each do |card|
+            if card.rank >= 11
+                high_rank_cards << card
+            end
+        end
+
+        high_rank_cards
+
+    end
+
+
+    def percent_high_ranking
+
+        
+        numerator = high_ranking_cards.length() * 1.00
+
+        denominator = @cards.length() * 1.00
```

Comment:
```suggestion
        numerator = high_ranking_cards.length.to_f

        denominator = @cards.length.to_f
```

In Ruby we don't need to include empty parens when there are no arguments. It is convention to these those off. It's also preferred to use `.to_f` which converts an integer to a float instead of multiplying by 1.00

Reply:
I'd realized that it wasn't needed later on in my codes but missed removing it here. The to_f method is something I got more exposure to after the project, so this makes a lot of sense now and definitely looks cleaner to me!